docs/How_to_contribute.md

```diff
@@ -64,3 +64,25 @@ A new version number shall be determined as follows:
 * Release adds a major new feature or modifies any interface (for example by modifying the format of input or output data) in such a way that it is <b>not fully compatible</b> with the previous version anymore.
     * ==> Increase first digit by `1`.
     * ==> Set remaining digits to `0`.
+
+# Developer hints
+## Output folder structure
+All tools developed as part of the DiscoPop Project make use of the following folder structure:
+```
+- project root/
+ - .discopop/
+  - common_data/
```

Comment:
I am not completely happy with the "common_data" directory. It adds another hierarchy level which is somewhat unspecified.

Two possible changes:
- simply drop the "common files" into .discopop without another hierarchy level
- interpret the filemapping script as a tool, too --> .discopop/filemapping/FileMapping.txt

Reply:
todo: move FileMapping.txt to .discopop

Reply:
todo: update tests